storage/reads/flux_table.gen.go.tmpl

```diff
@@ -0,0 +1,945 @@
+package reads
```

Comment:
```
$ git diff master:./storage/flux/table.gen.go.tmpl ./storage/reads/flux_table.gen.go.tmpl
diff --git a/storage/flux/table.gen.go.tmpl b/storage/reads/flux_table.gen.go.tmpl
index e6c168967c..84e0a5094c 100644
--- a/storage/flux/table.gen.go.tmpl
+++ b/storage/reads/flux_table.gen.go.tmpl
@@ -1,4 +1,4 @@
-package storageflux 
+package reads
 
 import (
 	"fmt"
@@ -12,11 +12,10 @@ import (
 	"github.com/influxdata/flux/interval"
 	"github.com/influxdata/flux/memory"
 	"github.com/influxdata/flux/values"
-	"github.com/influxdata/influxdb/v2/kit/platform/errors"
-	"github.com/influxdata/influxdb/v2/models"
-	"github.com/influxdata/influxdb/v2/storage/reads/datatypes"
-	storage "github.com/influxdata/influxdb/v2/storage/reads"
-	"github.com/influxdata/influxdb/v2/tsdb/cursors"
+	"github.com/influxdata/influxdb/kit/platform/errors"
+	"github.com/influxdata/influxdb/models"
+	"github.com/influxdata/influxdb/storage/reads/datatypes"
+	"github.com/influxdata/influxdb/tsdb/cursors"
 )
 {{range .}}
 //
@@ -163,8 +162,8 @@ func (t *{{.name}}WindowTable) createNextBufferTimes() (start, stop *array.Int64
 
 		// Create a buffer with the buffer size.
 		// TODO(jsternberg): Calculate the exact size with max points as the maximum.
-		startB.Resize(storage.MaxPointsPerBlock)
-		stopB.Resize(storage.MaxPointsPerBlock)
+		startB.Resize(MaxPointsPerBlock)
+		stopB.Resize(MaxPointsPerBlock)
 		for ; ; t.windowBounds = t.window.NextBounds(t.windowBounds) {
 			startT, stopT := t.getWindowBoundsFor(t.windowBounds)
 			if startT >= int64(t.bounds.Stop) {
@@ -323,7 +322,7 @@ func new{{.Name}}WindowSelectorTable(
 	done chan struct{},
 	cur cursors.{{.Name}}ArrayCursor,
 	bounds execute.Bounds,
-	window interval.Window, 
+	window interval.Window,
 	timeColumn string,
 	key flux.GroupKey,
 	cols []flux.ColMeta,
@@ -443,7 +442,7 @@ func new{{.Name}}EmptyWindowSelectorTable(
 		arr: cur.Next(),
 		rangeStart:  rangeStart,
 		rangeStop:   rangeStop,
-		windowBounds: window.GetLatestBounds(values.Time(rangeStart)),
+        windowBounds: window.GetLatestBounds(values.Time(rangeStart)),
 		window: window,
 		timeColumn:  timeColumn,
 	}
@@ -462,7 +461,7 @@ func (t *{{.name}}EmptyWindowSelectorTable) advance() bool {
 	}
 
 	values := t.arrowBuilder()
-	values.Resize(storage.MaxPointsPerBlock)
+	values.Resize(MaxPointsPerBlock)
 
 	var cr *colReader
 
@@ -492,7 +491,7 @@ func (t *{{.name}}EmptyWindowSelectorTable) advance() bool {
 
 func (t *{{.name}}EmptyWindowSelectorTable) startTimes(builder *array.{{.ArrowType}}Builder) *array.Int64 {
 	start := arrow.NewIntBuilder(t.alloc)
-	start.Resize(storage.MaxPointsPerBlock)
+	start.Resize(MaxPointsPerBlock)
 
 	for int64(t.windowBounds.Start()) < t.rangeStop {
 		// The first window should start at the
@@ -530,7 +529,7 @@ func (t *{{.name}}EmptyWindowSelectorTable) startTimes(builder *array.{{.ArrowTy
 			t.idx = 0
 		}
 
-		if start.Len() == storage.MaxPointsPerBlock {
+		if start.Len() == MaxPointsPerBlock {
 			break
 		}
 	}
@@ -539,7 +538,7 @@ func (t *{{.name}}EmptyWindowSelectorTable) startTimes(builder *array.{{.ArrowTy
 
 func (t *{{.name}}EmptyWindowSelectorTable) stopTimes(builder *array.{{.ArrowType}}Builder) *array.Int64 {
 	stop := arrow.NewIntBuilder(t.alloc)
-	stop.Resize(storage.MaxPointsPerBlock)
+	stop.Resize(MaxPointsPerBlock)
 
 	for int64(t.windowBounds.Start()) < t.rangeStop {
 		// The last window should stop at the end of
@@ -577,7 +576,7 @@ func (t *{{.name}}EmptyWindowSelectorTable) stopTimes(builder *array.{{.ArrowTyp
 			t.idx = 0
 		}
 
-		if stop.Len() == storage.MaxPointsPerBlock {
+		if stop.Len() == MaxPointsPerBlock {
 			break
 		}
 	}
@@ -586,13 +585,13 @@ func (t *{{.name}}EmptyWindowSelectorTable) stopTimes(builder *array.{{.ArrowTyp
 
 func (t *{{.name}}EmptyWindowSelectorTable) startStopTimes(builder *array.{{.ArrowType}}Builder) (*array.Int64, *array.Int64, *array.Int64) {
 	start := arrow.NewIntBuilder(t.alloc)
-	start.Resize(storage.MaxPointsPerBlock)
+	start.Resize(MaxPointsPerBlock)
 
 	stop := arrow.NewIntBuilder(t.alloc)
-	stop.Resize(storage.MaxPointsPerBlock)
+	stop.Resize(MaxPointsPerBlock)
 
 	time := arrow.NewIntBuilder(t.alloc)
-	time.Resize(storage.MaxPointsPerBlock)
+	time.Resize(MaxPointsPerBlock)
 
 	for int64(t.windowBounds.Start()) < t.rangeStop {
 
@@ -641,7 +640,7 @@ func (t *{{.name}}EmptyWindowSelectorTable) startStopTimes(builder *array.{{.Arr
 			t.idx = 0
 		}
 
-		if time.Len() == storage.MaxPointsPerBlock {
+		if time.Len() == MaxPointsPerBlock {
 			break
 		}
 	}
@@ -653,13 +652,13 @@ func (t *{{.name}}EmptyWindowSelectorTable) startStopTimes(builder *array.{{.Arr
 type {{.name}}GroupTable struct {
 	table
 	mu     sync.Mutex
-	gc     storage.GroupCursor
+	gc     GroupCursor
 	cur    cursors.{{.Name}}ArrayCursor
 }
 
 func new{{.Name}}GroupTable(
 	done chan struct{},
-	gc storage.GroupCursor,
+	gc GroupCursor,
 	cur cursors.{{.Name}}ArrayCursor,
 	bounds execute.Bounds,
 	key flux.GroupKey,
@@ -920,7 +919,7 @@ func (t *{{.name}}GroupTable) advanceCursor() bool {
 				Err: &GroupCursorError {
 					typ: "{{.name}}",
 					cursor: cur,
-				},
+                },
 			}
 			return false
 		} else {

```

Reply:
A few whitespace diffs are annoying.

Reply:
Also it's worrying that this `tmpl` didn't exist but the generated code was here?

Reply:
Whitespace differences addressed